addons/hr_homeworking_calendar/static/src/calendar/common/calendar_common_renderer.js

```diff
@@ -84,7 +84,10 @@ patch(AttendeeCalendarCommonRenderer.prototype, {
             const box = info.el.querySelector(`.fc-daygrid-day-top`);
             if (!box)
                 return;
-            const content = renderToString(this.constructor.ButtonWorklocationTemplate, this.headerTemplateProps(info.date));
+            const content = renderToString(this.constructor.ButtonWorklocationTemplate, {
```

Comment:
This fix should be implemented in the appointment app, not here. Could you please try that?